portfolio/src/main/webapp/script.js

```diff
@@ -29,6 +29,27 @@ function addRandomShow() {
 
 async function getName() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('hello-container').innerText = quote;
+  const name = await response.text();
+  document.getElementById('hello-container').innerText = name;
+}
+function getDataStats() {
+  fetch('/data').then(response => response.json()).then((stats) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
```

Comment:
You have this comment, which I think is correct, but below `stats` gets passed to a function expecting some text. Does that work as you expect it to?